python/ray/cloudpickle/cloudpickle_fast.py

```diff
@@ -192,13 +192,20 @@ def _class_getstate(obj):
 
         registry = clsdict.pop('_abc_registry', None)
         if registry is None:
-            # in Python3.7+, the abc caches and registered subclasses of a
-            # class are bundled into the single _abc_impl attribute
-            clsdict.pop('_abc_impl', None)
-            (registry, _, _, _) = abc._get_dump(obj)
-
-            clsdict["_abc_impl"] = [subclass_weakref()
-                                    for subclass_weakref in registry]
+            if hasattr(abc, '_get_dump'):
```

Comment:
This restores a previous cloudpickle fix for 3.6 compatibility: https://github.com/ray-project/ray/commit/dde49b8d48b0203d8230858b8c252b223628c160